### DISCUSSION TITLE
Horizontal Data Representation

### DISCUSSION THREAD

**Discussion by architpuri** [ozlerhakan/poiji#194] [Q&A] [answered]
Does poiji support conversion for data in horizontal format? Basically headers in column A

![image](https://user-images.githubusercontent.com/29012560/110940802-7d297c00-835d-11eb-82ea-7ae3dbfd079e.png)

**Comment by ozlerhakan** [comment, marked as answer]
Hi @architpuri ,

Poiji does not support this case. Feel free to implement it :)

**Reply by architpuri** [reply]
Sure would love to contribute.
